Library/Formula/mkcramfs.rb

```diff
@@ -0,0 +1,36 @@
+class Mkcramfs < Formula
+  homepage "http://cramfs.sourceforge.net/"
+  url "https://downloads.sourceforge.net/sourceforge/cramfs/cramfs-1.1.tar.gz"
+  sha1 "08133f2a2a0f02c6dd07c8dc565a4eac6bc967cd"
+
+  def install
+    system "make", "mkcramfs"
+    bin.install "mkcramfs"
+  end
+
+  test do
+    system "#{bin}/mkcramfs", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!